package.json

```diff
@@ -11,6 +11,7 @@
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
     "@types/lodash": "^4.14.165",
+    "@types/lodash.debounce": "^4.0.6",
```

Comment:
it should already be present in "@types/lodash

Reply:
fixed